### PR TITLE
Added travis file

### PR DESCRIPTION
**Pull request by jmulford-bw**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: node_js
```
Comment (andyfeller) [inline .travis.yml]: Thought the trial TravisCI account expired and different directions were being explored elsewhere

Comment (jmulford-bw) [reply .travis.yml]: Hmm. I'm not aware of any of this. I'm not sure if devx was even using the trial TravisCI account, we may have had our own as a legacy of when we were in product

Comment (jmulford-bw) [reply .travis.yml]: We could also be using a free account, we only use travis for public repos as far as I know

Comment (andyfeller) [reply .travis.yml]: Thought that might be the case but checking.

Comment (jmulford-bw) [reply .travis.yml]: 👍

**Pull request merged**